site/tests/app/libraries/IniParserTester.php

```diff
@@ -3,13 +3,101 @@
 namespace tests\app\libraries;
 
 use app\libraries\IniParser;
+use PHPUnit\Runner\Exception;
 
 class IniParserTester extends \PHPUnit\Framework\TestCase {
+    /**
+     * Test that an array writes correctly and is then read back exactly the same
+     * @param $writeData
+     * @param string $message
+     */
+    private static function assertReadWriteEquals($writeData, $message = "") {
```

Comment:
Remove the static as there's no real point to having it here. You can also get rid of the second parameter as it's unused?